kcat/Dockerfile.ubi8

```diff
@@ -67,6 +68,6 @@ RUN echo "Installing runtime dependencies for SSL and SASL support ...." \
 
 USER appuser
 
-RUN kafkacat -V
+RUN kcat -V
 
-CMD ["kafkacat"]
+CMD ["kcat"]
```

Comment:
I think this would be a good time to fix these images so they have kcat as an ENTRYPOINT, avoid the need to specify the executable name.
E.g.;:
`docker run -it cp-kcat kcat -V` (bad)
vs
`docker run -it cp-kcat -V` (good)

Reply:
My comment here is that this changes behavior in existing releases, so we shouldn't do it with existing releases (When its targeting 5.4.0-post), if this is for CP 7.1 and above, then 👍 to that change, but please follow-up with a note on this change in docs-platform/7.1.x so the change is communicated to our customers.

Reply:
We're changing the image name, which is a breaking change anyway, right?

Reply:
Fair point, then this change should target CP 7.1 (7.1.x) or above (master for CP 7.2)

Reply:
Done.